test/helpers/jobs.ts

```diff
@@ -15,7 +15,7 @@ import { RecordConstructor } from '../../app/models/record';
 export const adminUsername = 'adam';
 
 export const expectedJobKeys = [
-  'username', 'status', 'message', 'progress', 'createdAt', 'updatedAt', 'links', 'request', 'numInputGranules', 'jobID',
+  'username', 'status', 'message', 'progress', 'createdAt', 'updatedAt', 'links', 'request', 'numInputGranules', 'jobID', 'dataExpiration',
```

Comment:
I know this is the test, but for the source code as well I think we should change the order of the fields so we have all the times together `createdAt`, `updatedAt`, and then `dataExpiration`.

Reply:
Done